websites/W/WINDOWS93/iframe.ts

```diff
@@ -0,0 +1,254 @@
+const iframe = new iFrame();
+
+let minesweeper = {
+		time: 0,
+		bombs: 0,
+		gameState: "",
+	},
+	bananamp = {
+		title: "",
+		playing: false,
+	},
+	defrag = {
+		progress: "",
+		playing: false,
+		started: false,
+	},
+	solitude = {
+		stack1: "",
+		stack2: "",
+		stack3: "",
+		stack4: "",
+	},
+	maze = {
+		distance: "",
+	},
+	wlc = {
+		currentTime: 0,
+		duration: 0,
+		paused: false,
+	},
+	active = false;
+
+window.onfocus = function () {
+	active = true;
+};
+
+window.onblur = function () {
+	active = false;
+};
```

Comment:
Perhaps these could use `addEventListener` instead of overwriting the global event listener methods just in case?